tests/accuracy/test_sbs.py

```diff
@@ -59,7 +60,8 @@ class TestSbSLHC:
         Phase: TestCfg(1e-2, 1e-8, 5e-4, "phase"),
         BetaPhase: TestCfg(9e-2, 1e-10, 5e-4, "beta_phase"),
         AlphaPhase: TestCfg(1e-1, 1e-8, 8e-2, "alpha_phase"),
-        Dispersion: TestCfg(1e-2, None, None, "dispersion"),  # not really working at the moment, see https://github.com/pylhc/omc3/issues/498
+        Dispersion: TestCfg(1e-6, 1e-4, 1e-4, "dispersion"),  # not really working at the moment, see https://github.com/pylhc/omc3/issues/498
+        DispersionMomentum: TestCfg(1e-2, None, None, "dispersion"),  # not really working at the moment, see https://github.com/pylhc/omc3/issues/498

```

Comment:
so adding the dispersion momentum will not fix that? That's still a different error?